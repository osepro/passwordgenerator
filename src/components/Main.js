import React, { Component } from "react";
import UserInput from "components/UserInput";

class Main extends Component {
	render() {
		return (
			<div>
				<h1>Password Generator</h1>
				<UserInput />
			</div>
		);
	}
}

export default Main;
