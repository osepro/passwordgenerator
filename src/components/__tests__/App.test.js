import React from "react";
import { mount } from "enzyme";
import App from "../../App";
import Main from "components/Main";

let wrapped;

beforeEach(() => {
	wrapped = mount(<App />);
});

afterEach(() => {
	wrapped.unmount();
});

it("contains all components", () => {
	expect(wrapped).toMatchSnapshot();
});
it("shows Main component", () => {
	expect(wrapped.find(Main).length).toEqual(1);
});
