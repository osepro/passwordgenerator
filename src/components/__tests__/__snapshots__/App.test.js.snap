// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contains all components 1`] = `ReactWrapper {}`;
